index.html

```diff
@@ -1,111 +1,579 @@
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    <div id="container" class="container">
-        <div class="profile-folder">
-          <div class="folder">
-            <div class="title">
-              Yaba Kawaii Studio
-              <span class="title-btn">
-                <span class="btn1"></span>
-                <span class="btn2"></span>
-                <span class="btn3"></span>
-              </span>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bakbak+One&family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="all.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home page</title>
+  </head>
+  <body>
+    <header>
+      <a class="logo" href="#">Binabox</a>
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a href="#" class="nav-link">Home</a>
+          </li>
+          <li>
+            <a href="Blog.html" class="nav-link">Blog</a>
+          </li>
+          <li>
+            <a href="Catalog.html" class="nav-link">Catalog</a>
+          </li>
+          <li>
+            <a href="Contact-us.html" class="nav-link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <input type="text" size="2" />
+        <button><a href="link" class="button">Discard</a></button>
+        <button><a href="link" class="button button-blue">Connect</a></button>
+      </div>
+    </header>
+    <main>
+      <section class="statistic">
+        <ul>
+          WITH BINABOX EXPLORE NFT COLLECTION
+          <li>We are tne best way to check the rarity on NTF collection</li>
+          <li>Total item<span class="bold-text">2240+</span></li>
+          <li>Profiles whitelisted<span class="bold-text">1000+</span></li>
+        </ul>
+        <button type="submit">Connect walet</button>
+        <button type="submit">whitelist now</button>
+      </section>
+      <section class="catalog">
+        <div>
+          <h2><span class="folder-icon"></span>Catalog</h2>
+          <div></div>
+          <a href="catalog.html" class="link">See all</a>
+          <button>&#8249;</button>
+          <button>&#8250;</button>
+        </div>
+        <div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="lable">NEW</span>
+            <ul>
+              <li><span class="bold-text">$199</span></li>
```

Comment:
In this case, it is semantically incorrect to use a list. It's better to use the 'span' tag

Reply:
done